src/API/src/db/occurrence/occurrence.service.test.ts

```diff
@@ -348,6 +348,32 @@ describe('Occurrence service', () => {
       );
     });
 
+    it('on bionomics true', async () => {
+      const result = await service.findOccurrences(
+        3,
+        10,
+        { bionomics: [true] },
+        { locationWindowActive: false },
+      );
+      expect(result.items).toEqual(expectedOccurrences);
+      expect(mockQueryBuilder.andWhere).toHaveBeenCalledWith(
+        expect.any(Brackets),
+      );
+    });
+
+    it('on bionomics false', async () => {
+      const result = await service.findOccurrences(
+        3,
+        10,
+        { bionomics: [false] },
+        { locationWindowActive: false },
+      );
+      expect(result.items).toEqual(expectedOccurrences);
+      expect(mockQueryBuilder.andWhere).toHaveBeenCalledWith(
+        expect.any(Brackets),
+      );
```

Comment:
Are these test actually testing anything? We're calling the same `expect`s on both,